lib/r10k/action/deploy/module.rb

```diff
@@ -45,10 +60,11 @@ def visit_source(source)
         end
 
         def visit_environment(environment)
-          if @opts[:environment] && (@opts[:environment] != environment.dirname)
-            logger.debug1(_("Only updating modules in environment %{opt_env} skipping environment %{env_path}") % {opt_env: @opts[:environment], env_path: environment.path})
+          requested_envs = @settings[:overrides][:environments][:requested_environments]
+          if !requested_envs.empty? && requested_envs.none? { |env| env != environment.dirname }
```

Comment:
I find the second part of this logic really hard to parse. Do we know there can only ever be one thing in `requested_environments`? If so, can we simplify this to take advantage of that? Otherwise, I think just `include?` might work?

Reply:
Very good point wrt `include?`! The flag for this action is a single `--environment` but I wanted to try and find any similarities between this and environment handling in `deploy/environment.rb` in case we can merge this functionality later.

Reply:
Yeah makes sense to keep treating it as a real list.